ldap/servers/slapd/modify.c

```diff
@@ -1207,7 +1207,7 @@ op_shared_allow_pw_change(Slapi_PBlock *pb, LDAPMod *mod, char **old_pw, Slapi_M
         slapi_pblock_set(pb, SLAPI_BACKEND, slapi_be_select(&sdn));
 
         /* Check if ACIs allow password to be changed */
-        if (!pw_is_pwp_admin(pb, pwpolicy) && (res = slapi_acl_check_mods(pb, e, mods, &errtxt)) != LDAP_SUCCESS) {
+        if (!pw_is_pwp_admin(pb, pwpolicy, PR_FALSE) && (res = slapi_acl_check_mods(pb, e, mods, &errtxt)) != LDAP_SUCCESS) {
```

Comment:
Shouldn't it be PWP_INCLUDE_ROOTDN rather than PR_FALSE ?

Reply:
Nice catch, it will be fixed